en_US/advanced/shared-subscriptions.md

```diff
@@ -58,24 +60,28 @@ Shared subscriptions prefixed with `$queue/` are shared subscriptions without gr
 
 EMQX Broker's shared subscription supports balancing strategy and distribution of Ack configuration:
 
-```bash
-# etc/emqx.conf
-
-# balancing strategy
-broker.shared_subscription_strategy = random
-
-# Applicable to QoS1 QoS2 messages, when enabled, message will be distributed to another group when one group is offline
-broker.shared_dispatch_ack_enabled = false
+```
+broker {
+  # balancing strategy
+  shared_subscription_strategy: random
+  
+  # Applicable to QoS1 QoS2 messages, when enabled,
+  # message will be distributed to another subscriber when selected subscriber is offline
+  shared_dispatch_ack_enabled = false
+}
 ```
 <!-- TODO 待确认 -->
 
-| Balancing strategy |             Description             |
-| :---------- | :--------------------------- |
-| random      | Select randomly among all subscribers |
-| round_robin | According to the order of subscription |
-| sticky      | Always sent to the last selected subscriber |
-| hash        | According to the hash value of the publisher ClientID |
+| Balancing strategy |             Description                                                                                                         |
+| :----------------- | :------------------------------------------------------------------------------------------------------------------------------ |
+| random             | Select randomly among all subscribers                                                                                           |
+| round_robin        | According to the order of subscription                                                                                          |
+| sticky             | Always sent to the last selected subscriber                                                                                     |
+| hash               | According to the hash value of the publisher ClientID                                                                           |
+| hash_clientid      | According to the hash value of the publisher ClientID                                                                           |
```

Comment:
I think [`hash`](https://github.com/emqx/emqx/blob/9de327263589e18c324af3642db07751d289378e/apps/emqx/src/emqx_shared_sub.erl#L77) is deprecated, maybe we could omit it?

Reply:
```suggestion
| hash_clientid      | According to the hash value of the publisher ClientID                                                                           |
```